h/static/scripts/annotator/config.js

```diff
@@ -21,9 +35,9 @@ function config(window_) {
     }
   }
 
-  var annotFragmentMatch = window_.location.hash.match(/^#annotations:(.*)/);
-  if (annotFragmentMatch) {
-    options.annotations = annotFragmentMatch[1];
+  var directLinkedID = annotationIDs.extractIDFromURL(window_.location.href);
```

Comment:
I'd suggest adding a comment here to explain that this is not the only place where direct link fragments are detected. If I were reading this code in the future, I'd be unlikely to guess that this is the "last resort" place to detect the fragment, and that the Chrome extension is already doing this.